motionplan/kinematic.go

```diff
@@ -36,6 +36,15 @@ func ComputePosition(model referenceframe.Frame, joints *pb.JointPositions) (spa
 // position of the end effector as a protobuf ArmPosition even when the arm is in an out of bounds state.
 // This is performed statelessly without changing any data.
 func ComputeOOBPosition(model referenceframe.Frame, joints *pb.JointPositions) (spatialmath.Pose, error) {
+	if joints == nil {
+		errMsg := "joint positions are nil, check that you are passing non-empty joint positions when writing your driver"
+		return nil, errors.New(errMsg)
```

Comment:
Making these error messages part of referenceframe/errors.go would be an improvement I think.